scripts/build.py

```diff
@@ -94,8 +94,7 @@
 
 JS_FILEPATHS_NOT_TO_BUILD = (
     os.path.join(
-        'core', 'templates', 'dev', 'head', 'expressions',
-        'expression-parser.service.js'),
+        'core', 'templates', 'dev', 'head', 'expressions', 'parser.js'),
```

Comment:
@ankita240796 also ptal at this change

Reply:
Just for context, prior to my changes, `expression-parser.service.js` was auto-generated by using `python -m scripts.create_expression_parser` and then manually changing some lines in order to make it an angular module. I decided to simplify this by generating the parser into `parser.js` and then importing the necessary functions into `expression-parser.service.js`. Since `parser.js` is now the auto-generated file, it must be ignored and not built.

Reply:
Sounds good.